<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1072</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UpAndAway</string>
  </property>
  <property name="windowIcon">
   <iconset theme="face-smile"/>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhMultiLine</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout_3">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Kundensuche</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>380</width>
        <height>200</height>
       </size>
      </property>
      <property name="title">
       <string>Kunde</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QTableWidget" name="tableWidget">
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <row>
          <property name="text">
           <string> </string>
          </property>
         </row>
         <row>
          <property name="text">
           <string> </string>
          </property>
         </row>
         <row>
          <property name="text">
           <string> </string>
          </property>
         </row>
         <row>
          <property name="text">
           <string> </string>
          </property>
         </row>
         <row>
          <property name="text">
           <string> </string>
          </property>
         </row>
         <row>
          <property name="text">
           <string> </string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Reise-ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Start der Reise</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Ende der Reise</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Reise</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QTextBrowser" name="KundeID">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>25</height>
          </size>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QTextBrowser" name="KundeName">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>25</height>
          </size>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>200</height>
       </size>
      </property>
      <property name="title">
       <string>Reise</string>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Reise-ID</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Buchungen</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QTableWidget" name="tableWidget_2">
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <row>
          <property name="text">
           <string> </string>
          </property>
         </row>
         <row>
          <property name="text">
           <string> </string>
          </property>
         </row>
         <row>
          <property name="text">
           <string> </string>
          </property>
         </row>
         <row>
          <property name="text">
           <string> </string>
          </property>
         </row>
         <row>
          <property name="text">
           <string> </string>
          </property>
         </row>
         <row>
          <property name="text">
           <string> </string>
          </property>
         </row>
         <row>
          <property name="text">
           <string> </string>
          </property>
         </row>
         <row>
          <property name="text">
           <string> </string>
          </property>
         </row>
         <row>
          <property name="text">
           <string> </string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Buchungstyp</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Start</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Ende</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Preis</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QTextBrowser" name="ReiseID">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>25</height>
          </size>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Buchungsdetails</string>
      </property>
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Buchungs-ID</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QTextBrowser" name="BuchungsID">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>25</height>
          </size>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="BuchungStart">
         <property name="maximum">
          <number>99999999</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Ende</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="BuchungEnde">
         <property name="maximum">
          <number>99999999</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Preis</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="BuchungPreis">
         <property name="maximum">
          <double>99999999.989999994635582</double>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="Flug">
          <attribute name="title">
           <string>Flugbuchung</string>
          </attribute>
          <layout class="QFormLayout" name="formLayout_7">
           <item row="0" column="0">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>Abreiseort</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>Ankunftsort</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>Fluglinie</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>Klasse</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="Abreiseort"/>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="Ankunftsort"/>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="Fluglinie"/>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="Klasse"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="Hotel">
          <attribute name="title">
           <string>Hotelbuchung</string>
          </attribute>
          <layout class="QFormLayout" name="formLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Hotel</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Stadt</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Zimmertyp</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="Hotel_2"/>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="Stadt"/>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="Zimmertyp"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="Wagen">
          <attribute name="title">
           <string>Mietwagenreservierung</string>
          </attribute>
          <layout class="QFormLayout" name="formLayout_6">
           <item row="0" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Abholort</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Rückgabeort</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Firma</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Fahrzeugklasse</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="Abholort"/>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="Rueckgabeort"/>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="Firma"/>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="Fahrzeugklasse"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="6" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="Speichern">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Speichern</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Abbrechen">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Abbrechen</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1072</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionEinlesen"/>
   </widget>
   <addaction name="menuDatei"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionEinlesen">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Einlesen</string>
   </property>
  </action>
  <action name="actionAusw_hlen">
   <property name="text">
    <string>Kunde</string>
   </property>
  </action>
  <action name="actionKundensuche">
   <property name="text">
    <string>Kundensuche</string>
   </property>
  </action>
  <action name="actionNach_ID_suchen">
   <property name="text">
    <string>Nach ID suchen...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
